src/components/WorkOrderDetails.js

```diff
@@ -31,7 +31,7 @@ import TimeLog from "./WorkOrder/TimeLog";
 import WorkSpecifications from "./WorkOrder/WorkSpecifications";
 import UploadImage from "./WorkOrder/UploadImage";
 import api from "../queries/api";
-import { workOrderFields } from "../queries/fields";
+import { workOrderFields as FIELDS } from "../queries/fields";
```

Comment:
![image](https://user-images.githubusercontent.com/37249039/74553136-501d5680-4f1c-11ea-95fa-1e44061067f2.png)
👏Makes this component a lot cleaner, and thanks for subbing out those hardcoded field references!